### PR TITLE
Teaser component

### PR DESCRIPTION
**Pull request by tinterweb**
Screenshot 2021-02-26 at 09 34 32

### CODE REVIEW

File: gatsby/pharmaspectra-public-website/src/components/teaser.js [hunk 1]
```diff
@@ -0,0 +1,18 @@
+import React from "react"
+import { useStaticQuery, graphql, Link } from "gatsby"
```
Comment (Nel-r) [inline gatsby/pharmaspectra-public-website/src/components/teaser.js:2]: Do we need this here?

File: gatsby/pharmaspectra-public-website/src/styledComponents/teaser.js [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+
+const ImageWrap = styled.div`
+  border: 8px solid var(--blue4);
+  border-radius: 50%;
+  width: 204px;
+  height: 204px;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  background: #f5f7ff;
```
Comment (Nel-r) [inline gatsby/pharmaspectra-public-website/src/styledComponents/teaser.js:25]: Is this a new colour variable?

**Review by Nel-r — CHANGES_REQUESTED** [review]

File: gatsby/pharmaspectra-public-website/src/components/teaser.js (hunk 1 above)
Comment (tinterweb) [reply gatsby/pharmaspectra-public-website/src/components/teaser.js:2]: Nope i'll remove

File: gatsby/pharmaspectra-public-website/src/styledComponents/teaser.js (hunk 2 above)
Comment (tinterweb) [reply gatsby/pharmaspectra-public-website/src/styledComponents/teaser.js:25]: ah yes I will add 👍

**Review by Nel-r — APPROVED** [review]

**Pull request merged**